test/org/redhatcop/util/inventory/hosts

```diff
@@ -0,0 +1,2 @@
+[seed-hosts]
+localhost
```

Comment:
Can we make this `localhost ansible_connection=local` and remove `--connection=local` from the README?

Reply:
yes